src/pages/en/tutorial/2-pages/2.md

```diff
@@ -167,4 +167,6 @@ To create content for pages built with `.astro` files, I write text within stand
 
 - [Markdown Cheat Sheet from The Markdown Guide](https://www.markdownguide.org/cheat-sheet/)  <Badge>external</Badge>
 
+- [What are browser developer tools? MDN](https://developer.mozilla.org/en-US/docs/Learn/Common_questions/What_are_browser_developer_tools)  <Badge>external</Badge> - Learn the basics of using browser dev tools.
```

Comment:
```suggestion
- [What are browser developer tools? MDN](https://developer.mozilla.org/en-US/docs/Learn/Common_questions/What_are_browser_developer_tools)  <Badge>external</Badge> 
```
Thanks for adding this! The other ones on the page here don't have any description, and the title is pretty self-explanatory, so I'm taking that out.